### PR TITLE
Card

### PR DESCRIPTION
**Pull request by sanjay-xdr**
added all types of card
demo :- https://loving-mestorf-0a3389.netlify.app/component/card/card

### CODE REVIEW

**Review by nikhil-sharma-nks — COMMENTED** [review]
Everything looks good to be merged👍

File: Component/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+.card-price {
+  font-size: 1.9rem;
+  font-weight: bold;
+}
+
+.card-strikeprice {
+  font-size: 1.9rem;
+  font-weight: lighter;
```
Comment (tushargoyal812) [inline Component/Card/card.css:62]: use number for font weight instead of lighter

File: Component/Card/card.html [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+        <link rel="stylesheet" href="../../Utility CSS/utility.css" />
+        <link rel="stylesheet" href="./card.css" />
+        <link rel="icon" href="../../img/logo.png" type="image/x-icon" />
+        <title>Card</title>
+    </head>
+    <body>
+        <header>
+            <nav class="container-flex flex-sb flex-wr">
+                <div class="header__nav">
```
Comment (tushargoyal812) [inline Component/Card/card.html:16]: use hyphen everywhere for class names if you you using hyphen.

**Review by tushargoyal812 — COMMENTED** [review]
take care of suggestions and looks good to merge.

**Pull request merged**